apps/pom.xml

```diff
@@ -863,7 +863,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-site-plugin</artifactId>
-                    <version>3.4</version>
```

Comment:
Does this plugin need to be updated for this to work?

Reply:
Reverted this change. This doesnt need to change for anything to work. This was done to supress the warnings from the plugin.validation.